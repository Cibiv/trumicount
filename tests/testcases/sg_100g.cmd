$TRUMICOUNT --input-umitools-group-out "$TESTCASES/sg_100g.q20.gout.bz2" \
	--paired --filter-strand-umis --umipair-sep '-' \
	--molecules 1 --threshold 24 \
	--output-counts sg_100g.q20.tab &&
diff sg_100g.q20.tab "$TESTCASES/sg_100g.q20.tab.expected" >/dev/null
