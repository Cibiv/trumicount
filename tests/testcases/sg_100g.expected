gene	n.umis	n.tot	efficiency	depth	loss	n.obs
AAC73112	10	10	0.82	72	0.026	10
AAC73113	34	35	0.82	71	0.026	34
AAC73114	14	14	0.82	71	0.026	14
AAC73115	18	18	0.82	71	0.026	18
AAC73117	2	2.1	0.82	72	0.026	2
AAC73119	154	158	0.82	74	0.026	154
AAC73120	11	11	0.82	72	0.026	11
AAC73121	2	2.1	0.82	72	0.026	2
AAC73125	907	931	0.82	74	0.026	907
AAC73126	123	126	0.82	70	0.026	123
AAC73130	7	7.2	0.82	72	0.026	7
AAC73134	61	63	0.82	77	0.026	61
AAC73136	33	34	0.82	70	0.026	33
AAC73137	185	190	0.82	69	0.026	185
AAC73138	12	12	0.82	72	0.026	12
AAC73139	32	33	0.82	68	0.026	32
AAC73140	17	17	0.82	71	0.026	17
AAC73141	10	10	0.82	72	0.026	10
AAC73142	4	4.1	0.82	72	0.026	4
AAC73143	59	61	0.82	71	0.026	59
AAC73144	115	118	0.82	70	0.026	115
AAC73150	2	2.1	0.82	72	0.026	2
AAC73151	2	2.1	0.82	72	0.026	2
AAC73157	2	2.1	0.82	72	0.026	2
AAC73158	7	7.2	0.82	70	0.026	7
AAC73159	4	4.1	0.82	72	0.026	4
AAC73160	6	6.2	0.82	73	0.026	6
AAC73161	4	4.1	0.82	72	0.026	4
AAC73162	30	31	0.82	73	0.026	30
AAC73163	30	31	0.82	72	0.026	30
AAC73164	66	68	0.82	71	0.026	66
AAC73165	61	63	0.82	76	0.026	61
AAC73166	2	2.1	0.82	72	0.026	2
AAC73169	1	1	0.82	72	0.026	1
AAC73170	32	33	0.82	72	0.026	32
AAC73171	6	6.2	0.82	73	0.026	6
AAC73174	4	4.1	0.82	72	0.026	4
AAC73175	4	4.1	0.82	72	0.026	4
AAC73176	2	2.1	0.82	72	0.026	2
AAC73178	5	5.1	0.82	71	0.026	5
AAC73179	4	4.1	0.82	72	0.026	4
AAC73180	7	7.2	0.82	70	0.026	7
AAC73182	2	2.1	0.82	72	0.026	2
AAC73183	15	15	0.82	72	0.026	15
AAC73184	6	6.2	0.82	72	0.026	6
AAC73185	4	4.1	0.82	72	0.026	4
AAC73186	2	2.1	0.82	72	0.026	2
AAC73189	4	4.1	0.82	72	0.026	4
AAC73191	22	23	0.82	73	0.026	22
AAC73192	17	17	0.82	71	0.026	17
AAC73193	18	18	0.82	71	0.026	18
AAC73194	6	6.2	0.82	72	0.026	6
AAC73195	28	29	0.82	74	0.026	28
AAC73196	28	29	0.82	73	0.026	28
AAC73197	42	43	0.82	71	0.026	42
AAC73198	15	15	0.82	72	0.026	15
AAC73199	20	21	0.82	69	0.026	20
AAC73200	5	5.1	0.82	70	0.026	5
AAC73201	11	11	0.82	74	0.026	11
AAC73202	19	20	0.82	71	0.026	19
AAC73203	13	13	0.82	72	0.026	13
AAC73204	23	24	0.82	72	0.026	23
AAC73205	16	16	0.82	70	0.026	16
AAC73206	27	28	0.82	70	0.026	27
AAC73207	88	90	0.82	72	0.026	88
AAC73208	24	25	0.82	68	0.026	24
AAC73209	99	102	0.82	72	0.026	99
AAC73210	2	2.1	0.82	72	0.026	2
AAC73212	4	4.1	0.82	72	0.026	4
AAC73213	6	6.2	0.82	73	0.026	6
AAC73214	8	8.2	0.82	72	0.026	8
ACO59989	23	24	0.82	75	0.026	23
